pkg/utils/utils.go

```diff
@@ -133,7 +133,7 @@ func ParseLines(lines []string, params map[string]interface{}, logger logrus.Fie
 			// Skip the quotes in the value if present
 			unquotedValue, err := strconv.Unquote(string(value))
 			if err != nil {
-				logger.WithError(err).Errorf("Failed to unquote value %v for key %v. Just store the original value string", value, key)
+				logger.WithError(err).Debugf("Failed to unquote value %v for key %v. Just store the original value string", value, key)
```

Comment:
This one should not be changed to debug.  It should be at least Infof.

Reply:
This gets logged multiple times(every 5mins when we read secret) and the logs get filled with them.

[Global]
insecure-flag = "true"
cluster-id = "domain-c8"
cnsregistervolumes-cleanup-intervalinmin = 720
[VirtualCenter "sc1-10-78-120-77.eng.vmware.com"]
user = "workload_storage_management-8642bd46-491d-4501-a8ba-615482d98faa@vsphere.local"
password = "YdS!}:e#I9|8dIYI?+>r"
datacenters = "datacenter-3"
port = "443"

It gets logged because `cnsregistervolumes-cleanup-intervalinmin` value does not have quotes. I think it's harmless.